//
// fn main() {
//     // Fix the error
//     let _arr = [1, 2, '3'];
//
//     println!("Success!");
// }

fn main() {
    // fix the error
    let _arr = [1, 2, 3];
}